# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: security/sandbox/chromium-shim/patches/after_update/patch_order.txt
# Commit: 80fff55bee77
# Full Hash: 80fff55bee77ccef9b884556668e69efded13bae
# Author: Bob Owen <bobowencode@gmail.com>
# Date: 2019-07-16 04:07:00
# Description:
#   Bug 1564899: Make CloseHandleWrapper CHECK a DCHECK on non-Nightly builds. r=handyman
#   
#   This is because we are hitting it frequently during PolicyBase::OnJobEmpty and
#   currently we can't work out how this can happen.
#   
# ==============================================================================

diff -r 262f4fd60398 -r 80fff55bee77 security/sandbox/chromium-shim/patches/after_update/patch_order.txt
--- a/security/sandbox/chromium-shim/patches/after_update/patch_order.txt	Mon Jul 15 21:13:28 2019 +0000
+++ b/security/sandbox/chromium-shim/patches/after_update/patch_order.txt	Mon Jul 15 17:19:17 2019 +0000
@@ -4,3 +4,4 @@
 consult_PermissionsService_for_file_access.patch
 allow_flash_temporary_files.patch
 arm64_set_LoaderThreads.patch
+change_to_DCHECK_in_CloseHandleWrapper.patch